deploy/manifests/dev/us-east-2/patch.yaml

```diff
@@ -12,8 +12,8 @@ spec:
           args:
             - --port=8080
             - --indexer=https://cid.contact
-            - --sources=https://dev.cid.contact,https://cid.contact
-            - --targets=https://ago.dev.cid.contact,https://inga.prod.cid.contact
+            - --sources=https://dev.cid.contact,https://cid.contact,https://dev.cid.contact,https://dev.cid.contact,https://dev.cid.contact
+            - --targets=https://ago.dev.cid.contact,https://inga.prod.cid.contact,https://alva.dev.cid.contact,https://bria.dev.cid.contact,https://cora.dev.cid.contact
```

Comment:
Does this measure lag correctly for providers spread across multiple nodes? the new nodes are managed using assigner, which means each are responsible for ingesting from a portion of providers.

Reply:
It will measure distance for only those providers that are present at both indexers, so we should be alright I think